docs/api/IoT.js-API-BLE.md

```diff
@@ -9,64 +9,50 @@ The following shows ble module APIs available for each platform.
 | ble.setServices | O | O | X |
 
 
-### Contents
+# BLE - Bluetooth Low Energy
 
-- [BLE](#ble)
-    - [Module Functions](#module-functions)
-        - [`ble.startAdvertising(name, serviceUuids[, callback])`](#blestartadvertisingname-serviceuuids-callback)
-        - [`ble.stopAdvertising(callback)`](#blestopadvertisingcallback)
-        - [`ble.stopAdvertising(callback)`](#blestopadvertisingcallback-1)
-        - [`ble.setServices(services[, callback])`](#blesetservicesservices-callback)
-    - [Events](#events)
-        - [`'stateChange'`](#statechange)
-        - [`'advertisingStart'`](#advertisingstart)
-
-
-# BLE
-
-
-## Module Functions
-
-
-### `ble.startAdvertising(name, serviceUuids[, callback])`
-* `name <String>` - maximum 26 bytes
-* `serviceUuids <Array>`
+### ble.startAdvertising(name, serviceUuids[, callback])
+* `name` {string} - maximum 26 bytes
+* `serviceUuids` {[String]}
```

Comment:
Looks unclear for me. I put my suggestion, `{Array.<string>}` in #873. Let me know your thought.